environment.yml

```diff
@@ -0,0 +1,61 @@
+name: ribbit-dash-env
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - brotli=1.0.9=h8ffe710_5
+  - brotli-bin=1.0.9=h8ffe710_5
+  - brotli-python=1.0.9=py39h415ef7b_5
+  - ca-certificates=2021.5.30=h5b45459_0
+  - certifi=2021.5.30=py39hcbf5309_0
+  - click=8.0.1=py39hcbf5309_0
+  - colorama=0.4.4=pyh9f0ad1d_0
+  - dash=1.21.0=pyhd8ed1ab_0
+  - dash-core-components=1.17.1=pyhd8ed1ab_0
+  - dash-html-components=1.1.4=pyhd8ed1ab_0
+  - dash-renderer=1.9.1=pyhd8ed1ab_0
+  - dash-table=4.12.0=pyhd8ed1ab_0
+  - dataclasses=0.8=pyhc8e2a94_3
+  - flask=2.0.1=pyhd8ed1ab_0
+  - flask-compress=1.10.1=pyhd8ed1ab_0
+  - future=0.18.2=py39hcbf5309_3
+  - intel-openmp=2021.3.0=h57928b3_3372
+  - itsdangerous=2.0.1=pyhd8ed1ab_0
+  - jinja2=3.0.1=pyhd8ed1ab_0
+  - libblas=3.9.0=11_win64_mkl
+  - libbrotlicommon=1.0.9=h8ffe710_5
+  - libbrotlidec=1.0.9=h8ffe710_5
+  - libbrotlienc=1.0.9=h8ffe710_5
+  - libcblas=3.9.0=11_win64_mkl
+  - liblapack=3.9.0=11_win64_mkl
+  - markupsafe=2.0.1=py39hb82d6ee_0
+  - mkl=2021.3.0=hb70f87d_564
+  - numpy=1.21.2=py39h6635163_0
+  - openssl=1.1.1k=h8ffe710_1
+  - pandas=1.3.2=py39h2e25243_0
+  - pip=21.2.4=pyhd8ed1ab_0
+  - plotly=5.3.1=pyhd8ed1ab_0
+  - python=3.9.7=h7840368_0_cpython
+  - python-dateutil=2.8.2=pyhd8ed1ab_0
+  - python_abi=3.9=2_cp39
+  - pytz=2021.1=pyhd8ed1ab_0
+  - pyyaml=5.4.1=py39hb82d6ee_1
+  - setuptools=57.4.0=py39hcbf5309_0
+  - six=1.16.0=pyh6c4a22f_0
+  - sqlite=3.36.0=h8ffe710_0
+  - tbb=2021.3.0=h2d74725_0
+  - tenacity=8.0.1=pyhd8ed1ab_0
+  - tzdata=2021a=he74cb21_1
+  - ucrt=10.0.20348.0=h57928b3_0
+  - vc=14.2=hb210afc_5
+  - vs2015_runtime=14.29.30037=h902a5da_5
+  - werkzeug=2.0.1=pyhd8ed1ab_0
+  - wheel=0.37.0=pyhd8ed1ab_1
+  - yaml=0.2.5=he774522_0
+  - pip:
+    - gunicorn==20.1.0
+    - influxdb-client==1.20.0
+    - rx==3.2.0
+    - urllib3==1.26.6
+prefix: C:\Users\steve\Anaconda3\envs\ribbit-dash-env
```

Comment:
Looks like this would break on other people's computers... it's probably not a good idea to commit this.

What's the benefit of including this file *and* the `requirements.txt` file?

Reply:
Good catch @brianstrauch !

Reply:
Thanks @brianstrauch ! I can fix this and add conda instructions. I guess the benefit for including both `requirements.txt` and `environment.yml` is to easily support both venv and conda environment formats?